src/system/components/handler.clj

```diff
@@ -1,7 +1,8 @@
 (ns system.components.handler
   (:require [com.stuartsierra.component :as component]
             [lang-utils.core :refer [contains+? ∘]]
-            [compojure.core :as compojure]))
+            [compojure.core :as compojure]
+            [bidi.ring :as bidi]))
```

Comment:
This forces all users to have bidi as a dependency, even if they are never going to use it. We don't want to do that. Please resolve dynamically later. For example,
`(ns-resolve 'bidi.ring (symbol "make-handler"))`

Reply:
Fixed.